client-device-connection-base/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2024 Contributors to the Eclipse Foundation
+ 
+    See the NOTICE file(s) distributed with this work for additional
+    information regarding copyright ownership.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0
+
+    SPDX-License-Identifier: EPL-2.0
+ -->
+
+<!DOCTYPE configuration>
+
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <root level="INFO">
+    <appender-ref ref="STDOUT" />
+  </root>
+
+  <logger name="org.eclipse.hono.client" level="INFO"/>
+  <logger name="org.eclipse.hono.config" level="INFO"/>
+  <logger name="org.eclipse.hono.connection" level="INFO"/>
+  <logger name="org.eclipse.hono.util" level="INFO"/>
```

Comment:
given that this is the logging config for tests in package `org.eclipse.hono.deviceconnection.common`, I would expect to also find some definition(s) for that package ...